src/microbe_stage/Microbe.cs

```diff
@@ -1732,6 +1742,9 @@ private Vector3 DoBaseMovementForce(float delta)
             force *= 0.5f;
         }
 
+        if (IsPlayerMicrobe)
+            force *= (float)CheatManager.Speed;
```

Comment:
Why is the cast to float here?

Reply:
`Speed` is a double and `force` is a float

Reply:
`Speed` is a double, because the godot slider gives me a double. I could also do the cast to float in the method, where godot gives me a new slider value.

Reply:
Then set speed as a float as well? Everything should be a float by default, and using a double needs special justification.

Reply:
I'm pretty sure elsewhere we just use a "float" as the parameter in a slider callback and it just works.

Reply:
I've oriented myself at the godot tooltip at the signal and haven't tried this. Thanks